### PR TITLE
fix: issues with kcat guide

### PR DESCRIPTION
**Pull request by wtrocki**
Reports have been made about missing binary name and also kcat not working at all (no messages produced on mac).
We need to make this guide functional for the summit. Any improvements appreciated.

### CODE REVIEW

File: docs/kafka/kcat-kafka/README.adoc [hunk 1]
```diff
@@ -164,6 +166,9 @@ $ kafkacat -t my-first-kafka-topic -b "$KAFKA_HOST" \
 
 NOTE: {product-kafka} also supports the SASL/OAUTHBEARER mechanism for authentication, which is the recommended authentication mechanism to use. However, Kafkacat does not yet fully support OAUTHBEARER, so this example uses SASL/PLAIN.
 
+NOTE: kafkacat producer might behave differently depending on the operating system. For Mac operating systems 
+messages can be sent using pipe. For example "echo "message" | kcat ... "
```
Comment (wtrocki) [inline docs/kafka/kcat-kafka/README.adoc:170]:
```suggestion
NOTE: kafkacat producer might behave differently depending on the operating system. For Mac operating systems messages can be sent by redirecting them to the kafkacat binary. For example "echo "message" | kafkacat ... " 
```

File: docs/kafka/kcat-kafka/README.adoc [hunk 2]
```diff
@@ -166,8 +166,7 @@ $ kafkacat -t my-first-kafka-topic -b "$KAFKA_HOST" \
 
 NOTE: {product-kafka} also supports the SASL/OAUTHBEARER mechanism for authentication, which is the recommended authentication mechanism to use. However, Kafkacat does not yet fully support OAUTHBEARER, so this example uses SASL/PLAIN.
 
-NOTE: kafkacat producer might behave differently depending on the operating system. For Mac operating systems 
-messages can be sent using pipe. For example "echo "message" | kcat ... " 
+NOTE: kafkacat producer might behave differently depending on the operating system. For Mac operating systems messages can be sent by redirecting them to the kafkacat binary. For example "echo "message" | kafkacat ... "
```
Comment (sterobin) [inline docs/kafka/kcat-kafka/README.adoc:169]:
```suggestion
NOTE: A Kafkacat producer might behave differently depending on the operating system. For example, on Mac operating systems, messages are sent by redirecting them to the Kafkacat binary in the format `echo "message" | kafkacat ...`.
```

**Comment by wtrocki** [comment]
FYI @DuncanDoyle

**Comment by wtrocki** [comment]
@rkpattnaik780 for tech approval.

**Review by rkpattnaik780 — APPROVED** [review]

**Pull request merged**